packages/suite/src/types/suite/guide.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+export type GuideView = 'GUIDE_DEFAULT' | 'GUIDE_CATEGORY' | 'GUIDE_ARTICLE';
+export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBACK_SUGGESTION';
+export type ActiveView = GuideView | FeedbackView;
+export type Category =
+    | 'dashboard'
+    | 'account'
+    | 'settings'
+    | 'send'
+    | 'receive'
+    | 'trade'
+    | 'other';
+export type Rating = 1 | 2 | 3 | 4 | 5;
```

Comment:
Can we add a comment explaning what the numbers mean? Like
```suggestion
// 5 is best, 1 is worst
export type Rating = 1 | 2 | 3 | 4 | 5;
```